### PR TITLE
Fix discord button

### PR DESCRIPTION
**Pull request by AshwanthramKL**
## Fixes Issue
Fixes Issue #23 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

### CODE REVIEW

File: app/(pages)/(home)/page.tsx [hunk 1]
```diff
@@ -151,6 +150,7 @@ export default function Home() {
                 // target="_blank"
```
Comment (ManishBisht777) [inline app/(pages)/(home)/page.tsx]: no need to comment these

File: app/(pages)/(home)/page.tsx [hunk 2]
```diff
@@ -151,6 +150,7 @@ export default function Home() {
...
                 // rel="noreferrer"
                 className={cn(buttonVariants(), "px-4")}
```
Comment (ManishBisht777) [inline app/(pages)/(home)/page.tsx]: you can just add className along with `className={cn(buttonVariants(), "px-4")}`

like `className={cn(buttonVariants(), "px-4 tailwind-class-you-wan-to-add")}`

**Review by ManishBisht777 — COMMENTED** [review]
left some comments

File: app/(pages)/(home)/page.tsx (hunk 2 above)
Comment (AshwanthramKL) [reply app/(pages)/(home)/page.tsx]: Okay

Comment (AshwanthramKL) [reply app/(pages)/(home)/page.tsx]: Done

**Review by ManishBisht777 — APPROVED** [review]
Thanks for contributing @AshwanthramKL 

LGTM

**Pull request merged**